### PR TITLE
OCPBUGS#11224: Update to required access policies

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.12+

Issue:
This PR addresses [ocpbugs-11224](https://issues.redhat.com/browse/OCPBUGS-11224)

Link to docs preview:

[Required access policies](https://62272--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-account.html#required-access-policies-ibm-cloud_installing-ibm-cloud-account)

- [ ] QE has approved this change.

### CODE REVIEW

File: modules/installation-ibm-cloud-iam-policies-api-key.adoc [hunk 1]
```diff
@@ -41,6 +41,12 @@ You must assign the required access policies to your IBM Cloud account.
 |Editor, Operator, Viewer, Administrator
 |
 
+|Account management ^[2]^
```
Comment (cjschaef) [inline modules/installation-ibm-cloud-iam-policies-api-key.adoc:44]: This new policy is required when using an existing ResourceGroup and when IPI has to create a new ResourceGroup.
This requirement is necessary for IPI to lookup ResourceGroups during `manifest generation`, for `ccoctl` to create Secrets via CredentialsRequests, and other instances during cluster creation.

I recommend remove `[2]` from this policy, as it is always required (not optional).

I added some details to the Jira
https://issues.redhat.com/browse/OCPBUGS-11224?focusedId=22775393

Comment (mjpytlak) [reply modules/installation-ibm-cloud-iam-policies-api-key.adoc]: > I recommend remove `[2]` from this policy, as it is always required (not optional).

@cjschaef Thank you for the review. Removed the footnote. Please take one more look.

**Comment by mjpytlak** [comment]
@MayXuQQ Ready for QE review. PTAL. Thank you.

**Comment by cjschaef** [comment]
@MayXuQQ 
Due to a bug with `ccoctl` currently (requiring the need for [OCPBUGS_10741](https://issues.redhat.com/browse/OCPBUGS-10741)), `ccoctl` does not have logic to know if IBM CIS or IBM DNS Services is required for the Ingress CredentialsRequests (based on what is needed for a Public/CIS cluster or a Private/DNS cluster).

As far as the ResourceGroup IAM policy, yes, the issue you ran into is restricting your attempts since it is only for `Default` ResourceGroup. Going thru the UI to configure the proper ResourceGroup policy has been tricky. I can provide you details on what I have done to get the proper policy setup for the Service Id (or user). Prior to changes, it was straightforward to do so.

**Review by cjschaef — APPROVED** [review]

**Comment by MayXuQQ** [comment]
image
then added 2 policies
image
go ahead and delete/remove that All Identity and Access enabled services policy

**Comment by kcarmichael08** [comment]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**